src/Gen-ZAM.cc

```diff
@@ -1825,7 +1825,7 @@ void ZAM_InternalOpTemplate::Parse(const string& attr, const string& line, const
 				}
 		}
 
-	eval += "f->SetOnlyCall(z.call_expr);\n";
+	eval += "f->SetOnlyCall(z.call_expr.get());\n";
```

Comment:
Any reason to just not pass the `IntrusivePtr` directly to `SetOnlyCall` instead of the raw pointer?

Reply:
That would be changing code in the main part of Zeek  (`Frame.h`), used for Trigger associations. So doing this more cleanly would require more extensive changes, not just an API tweak, that's all.

Reply:
Ok, that makes sense.